src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java

```diff
@@ -162,4 +171,16 @@ public static void applyLastColorToFollowingLines(@NotNull List<String> lines) {
             }
         }
     }
+
+    public static String formatPrice(double value) {
+        Locale locale = new Locale(SimpleClans.getInstance().getSettingsManager().getString(ECONOMY_LOCALE_FORMAT));
+        DecimalFormatSymbols symbols = new DecimalFormatSymbols(locale);
```

Comment:
Not sure if this is needed since ECONOMY_DECIMAL_FORMAT_PATTERN is being used.

Reply:
He probably meant, DecimalFormatSymbols could define decimal separator and grouping separator by its pattern.

Reply:
Personally, I like the idea with decimal and group separator. Finding a right *locale* is kinda unintuitive.

Reply:
I prefer fetching it from the default locale.

Reply:
> I prefer fetching it from the default locale. 

You mean, JVM locale or plugin's

Reply:
Brazilian currency is supported.

Reply:
Try using `#,##0.00`.

Reply:
Now the problem is we need to split "pt_BR" into 2 strings.
Something like this:
```java
Locale loc = new Locale(lang.split("_")[0], lang.split("_")[1]));
```

(This is just a draft, it needs optimizing and safety checks, like array length.)